Wangscape/MetaOutput.h

```diff
@@ -21,14 +24,16 @@ class MetaOutput
     void writeTilesetData(std::string filename) const;
     void writeTerrainHypergraph(std::string filename) const;
     void writeAll(const Options& options) const;
-    const rapidjson::Document& getTileData() const;
-    const rapidjson::Document& getTileGroups() const;
-    const rapidjson::Document& getTilesetData() const;
+    const std::vector<Tile>& getTileData() const;
+    const std::unordered_map<std::string, std::vector<TileLocation>>& getTileGroups() const;
```

Comment:
Ok, I'll look for such type names and I'll move them to a separate files.

`unordered_map` has a bit better performance, thus, as far as I know, is better for lookup usage. The difference is that `unordered_map` needs hash function for custom keys, but that's not the case. I think that's additional argument for the typename.